### PR TITLE
add test conditional for Rollback

### PR DESCRIPTION
**Pull request by afragen**
Adding a conditional to `WP_Upgrader::install_package()` for testing of Rollback feature and Rollback feature plugin.

Trac ticket: https://core.trac.wordpress.org/ticket/56057

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Comment by afragen** [comment]
Now single line conditional.

File: src/wp-admin/includes/class-wp-upgrader.php [hunk 1]
```diff
@@ -593,7 +593,12 @@ public function install_package( $args = array() ) {
 		}
 
 		// Copy new version of item into place.
-		$result = copy_dir( $source, $remote_destination );
+		if ( class_exists( 'Rollback_Update_Failure\WP_Upgrader' ) && function_exists( '\Rollback_Update_Failure\move_dir' )
+		) {
```
Comment (jrfnl) [inline src/wp-admin/includes/class-wp-upgrader.php]:
```suggestion
		if ( class_exists( 'Rollback_Update_Failure\WP_Upgrader' ) && function_exists( '\Rollback_Update_Failure\move_dir' ) ) {
```

Either use a multi-line statement or truly use a single-line statement, this is sort of a mix of both and not allowed.

Comment (afragen) [reply src/wp-admin/includes/class-wp-upgrader.php]: oops. fixed. Thanks Juliette.

**Review by costdev — APPROVED** [review]
LGTM 👍 I've tested this locally and it works for me.

**Comment by SergeyBiryukov** [comment]
Thanks for the PR! Merged in https://core.trac.wordpress.org/changeset/53578.

**Pull request closed without merging**